ckan/templates/user/snippets/followee_dropdown.html

```diff
@@ -20,7 +20,7 @@
     <div class="popover-header">
       <div class="input-group">
         <span class="input-group-addon"><i class="fa fa-search"></i></span>
-        <input type="text" class="form-control" name="q" placeholder="{{ _('Search list...') }}" value="{{context.q}}" autocomplete="off">
+        <input type="text" class="form-control" name="q" placeholder="{{ _('Search list...') }}" value="{{context.q}}" autocomplete="off" aria-label="Search followee">
```

Comment:
One small issue I didn't notice before, could you add translation tags to the aria label, eg. _('Search followee') so it would be translated too if the user uses some other language than english.

Reply:
> One small issue I didn't notice before, could you add translation tags to the aria label, eg. _('Search followee') so it would be translated too if the user uses some other language than english.

Good point. In addition to adding a translation tag, I also extracted all messages and updated the entire dictionary.

Reply:
However, translations are extracted and compiled during release process as it makes harder to see what has actually changed in pull requests and produces unnecessary conflicts if PR's modify the compiled translations.

Reply:
> However, translations are extracted and compiled during release process as it makes harder to see what has actually changed in pull requests and produces unnecessary conflicts if PR's modify the compiled translations.

True, that's a good point. I guarantee I never modified the compiled translations though. How about now?

Reply:
Now it's all good :)